hawkular-wildfly-agent-feature-pack/src/main/resources/modules/system/layers/base/org/hawkular/agent/main/module.xml

```diff
@@ -83,6 +83,7 @@
     <module name="org.jboss.vfs"/>
 
     <!-- other dependencies we need -->
+    <module name="javaee.api"/> <!-- bus-commons -->
```

Comment:
> The agent's use of bus-common is only BasicMessage

Agent really does not use JMS? - Actually, you may well be right :)

In such a case, I'd say that the proper solution should consist in splitting `BasicMessage` & Co. from bus and putting it in a new artifact, call it e.g. `commons-messages`.

As an independent but related topic, we should have a look at the possibility to move message classes from `cmdgw` to the new `commons-messages`. That would allow Alerts to drop the dependency on `cmdgw`.